Cargo.toml

```diff
@@ -91,6 +91,7 @@ members = [
     "language/testing-infra/test-generation",
     "language/tools/disassembler",
     "language/tools/genesis-viewer",
+    "language/tools/libra-validator-interface",
```

Comment:
Does this libra-specific tool belong in `language/tools`? 

As we think about splitting Move into it's own repo, it seems easiest if all the tools are Move-related and not Libra-related (which I think is mostly true except for transaction-replay).

Reply:
Ahhh I was just thinking about putting thi new cratge inside the `tool` and didn't think about this complication. I would actually argue that it's the transaction-replay and the vm-genesis that is more libra-related than move related. Should we move both crates to a new folder under `language/`? OR `libra-vm/`

Reply:
Perhaps something like `language/libra-tools` or `libra/tools`? Not sure.